src/lem/tag.rs

```diff
@@ -0,0 +1,45 @@
+use std::fmt::Display;
+
+use crate::field::LurkField;
+
+#[derive(Clone, Copy, PartialEq, Debug, PartialOrd, Ord, Eq, Hash)]
+pub enum Tag {
+    // expression tags
+    Num,
+    U64,
+    Char,
+    Str,
+    Comm,
+    Fun,
+    Cons,
+    Sym,
+    Key,
+    // continuation tags
+    Outermost,
+    Dummy,
```

Comment:
Still here, though I think it moved. This comment is based on a previous exchange suggesting it was slated for removal — just checking my understanding.

Reply:
Sorry, I said that it would be removed because I thought it wasn't a real tag on `master`. But that tag does exist as a continuation tag so I think I shouldn't remove it